src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateActionPlanTest.kt

```diff
@@ -235,20 +233,6 @@ class CreateActionPlanTest : IntegrationTestBase() {
       .hasTitle(createStepRequest.title)
       .hasStatus(StepStatus.NOT_STARTED)
       .wasCreatedBy(dpsUsername)
-
-    // assert timeline event is created successfully
```

Comment:
I'm relying on the dedicated `GetTimelineTest` to test this functionality. Otherwise these `// Then` sections were becoming massive!